src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -133,17 +146,22 @@ export default class Pilot {
         const altitudeInstruction = radio_trend('altitude', currentAltitude, altitude);
         const altitudeVerbal = radio_altitude(readbackAltitude);
         let expediteReadback = '';
+        let altitudeUnattainable = '';
 
         if (expedite) {
             // including space here so when expedite is false there isnt an extra space after altitude
             expediteReadback = ' and expedite';
 
             this.shouldExpediteAltitudeChange();
         }
+        
+        if (this.assignedUnattainableAltitude) {
+            altitudeUnattainable = 'requested altitude unattainable, ';
+        }
 
         const readback = {};
-        readback.log = `${altitudeInstruction} ${readbackAltitude}${expediteReadback}`;
-        readback.say = `${altitudeInstruction} ${altitudeVerbal}${expediteReadback}`;
+        readback.log = `${altitudeUnattainable}${altitudeInstruction} ${readbackAltitude}${expediteReadback}`;
```

Comment:
I know we don't have any tests for the readbacks, in this method.  But, with the addition of another logic block, could you add one for when an altitude is attainable and when one is not?